services/frontend-service/src/ui/components/dropdown/checkbox.scss

```diff
@@ -15,7 +15,10 @@ along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>
 Copyright 2023 freiheit.com*/
 
 .checkbox-wrapper {
+    font-size: medium;
+    font-weight: normal;
     .mdc-button__label {
+        // this is the actual checkbox part of the checkbox component
         font-size: x-large;
```

Comment:
What would you suggest exactly?

Reply:
```suggestion
    @extend .text-regular; // has font-size: 12px
    font-size: medium;
    font-weight: normal;
    .mdc-button__label {
        // this is the actual checkbox part of the checkbox component
        @extend .headline1 // has font-size: 24px
```
Something like this. I'm not sure how big the change is from `medium` to `x-large` but this would achieve the increase in size. Thing is that it will also increase the line-height and the font-weight. If this is not desirable, maybe we should think about creating a new class like `text-large`

Reply:
it works! :)